rest/route/middleware.go

```diff
@@ -442,6 +458,10 @@ func urlVarsToDistroScopes(r *http.Request) (string, int, error) {
 	return distroID, http.StatusOK, nil
 }
 
+func superUserResource(_ *http.Request) (string, int, error) {
```

Comment:
This isn't needed if there's only 1 resource

Reply:
right, but it needs to somehow get the name of that resource

Reply:
The name of the resource is passed in to the HasPermission function and the appropriate scope is looked up by gimlet